lib/ansible/modules/cloud/digital_ocean/digital_ocean_droplet.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_droplet
+short_description: Create and delete a droplet
+description:
+     - Create and delete a droplet in DigitalOcean and optionally wait for it to be active.
+version_added: "2.5"
+author: "Gurchet Rai (@gurch101)"
+options:
+  state:
+    description:
+     - Indicate desired state of the target.
+    default: present
+    choices: ['present', 'absent']
+  id:
+    description:
+     - Numeric, the droplet id you want to operate on.
+  name:
+    description:
+     - String, this is the name of the droplet - must be formatted by hostname rules, or the name of a SSH key.
+  unique_name:
+    description:
+     - Bool, require unique hostnames.  By default, DigitalOcean allows multiple hosts with the same name.  Setting this to "yes" allows only one host
+       per name.  Useful for idempotence.
+    default: "no"
+    choices: [ "yes", "no" ]
+  size:
+    description:
+     - This is the slug of the size you would like the droplet created with.
+  image:
+    description:
+     - This is the slug of the image you would like the droplet created with.
+  region:
+    description:
+     - This is the slug of the region you would like your server to be created in.
+  ssh_key_ids:
+    description:
+     - Optional, array of SSH key (numeric) ID that you would like to be added to the server.
+  virtio:
+    description:
+     - Bool, turn on virtio driver in droplet for improved network and storage I/O.
+    default: "yes"
+    choices: [ "yes", "no" ]
+  private_networking:
+    description:
+     - Bool, add an additional, private network interface to droplet for inter-droplet communication.
+    default: "no"
+    choices: [ "yes", "no" ]
+  backups_enabled:
```

Comment:
I believe `backups_enabled`is duplicated in the doc string. Should this be removed since you listed `backups` on line 88 and use the backups parameter in the argument spec?